### ISSUE TITLE
The code is different with code generated in google's app<Authenticator> 

### ISSUE THREAD

**Issue by nemoleo** [Authenticator-Extension/Authenticator#460]
<Do not use this to ask about lost codes or accounts>
<More info here: https://authenticator.cc/docs/en/lost-codes>

**Describe the bug:** <Describe the issue with Authenticator> 
I was using the same QR Code to import into both google's mobile app & chrome plugin  <Authenticator>
but the code generated at the same time is different, only the code in the mobile app is able to pass the system verification.

 [app screenshot](http://chuantu.xyz/t6/729/1587440933x2728306394.jpg)
 [plugin screenshot](http://chuantu.xyz/t6/729/1587440962x989499252.jpg)

**Platform:** <iMac>
 - Browser: [Chrome]
 - Browser Version: 81.0.4044.113

**Comment by Sneezry** [comment]
Have you tried to sync clock with Google? You can do that by opening menu and click "Sync Clock with Google".

**Comment by nemoleo** [comment]
> Have you tried to sync clock with Google? You can do that by opening menu and click "Sync Clock with Google".

Yes, I 've tried this, but after sync done ,the code is still different with code in app

**Comment by jensMG** [comment]
@nemoleo, have you added a username to your plugin configuration manually?

**Comment by houtsnip** [comment]
I get the same issue, with Authenticator on Chrome producing different codes to Authenticator on Firefox.
Tried syncing that doesn't make any difference.
Anyway, for all other sites the codes generated by both extensions are the same.
The problem is for one particular site (lastpass.com).
The codes generated by the Chrome extension just don't work for this site.
I've tried exporting from both extensions and comparing them ... the secret is different!
Then I've tried exporting from Firefox and then importing into Chrome, but the same problem ... then I tried exporting from the Chrome extension (what was just imported from Firefox), and the Chrome extension has changed the secret, and it does this consistently ... that is, it consistently changes the secret to something else, which doesn't work.

**Comment by mymindstorm** [comment]
@houtsnip do you have encryption enabled on either installation of the extension?
